### PR TITLE
Updates for Page Builder preview for product and category forms

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) updates the user guide for the Page Builder preview for product and category descriptions, and the change of the short description for products displayed before the main description. It also includes edits for product and category content settings.

**_Internal staging build at: 547/user-guide/_**

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable: 2.4.1

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/categories-content-settings.html
https://docs.magento.com/user-guide/catalog/category-create.html
https://docs.magento.com/user-guide/catalog/product-content.html
https://docs.magento.com/user-guide/configuration/general/content-management.html

## Links to Magento issues or PRs

https://github.com/magento/magento2-page-builder/issues/511
https://github.com/magento/magento2-page-builder/issues/543
https://github.com/magento/magento2-page-builder/issues/544

whatsnew
Updated information for [product content](https://docs.magento.com/user-guide/catalog/product-content.html) and [category content](https://docs.magento.com/user-guide/catalog/categories-content-settings.html) for Page Builder preview support. Updated the [Advanced Content Tools](https://docs.magento.com/user-guide/configuration/general/content-management.html#advanced-content-tools) configuration to include the new _Enable Page Builder Content Preview_ setting.

### CODE REVIEW

**Comment by dshevtsov** [comment]
running tests

**Comment by dshevtsov** [comment]
running tests

File: src/catalog/categories-content-settings.md [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
-|Description|Store View|Specifies a description that appears on the category page. <br/><br/><span class="ee-only"></span><span class="btn">Edit with Page Builder</span> - Opens the [Page Builder workspace]({% link cms/page-builder-workspace.md %}), where you can edit the description. <br/><br/><span class="ee-only"></span><span class="btn">Show / Hide Editor</span>  - Toggles the display between WYSIWYG editor and HTML modes.|
-|Add CMS Block|Store View|Adds an existing [CMS block]({% link cms/blocks.md %}) to the category page.|
+|Category Image|Store View|Specifies an image for the top of the category page. Methods: <br/><br/><span class="btn">Upload</span> - Uploads an image file from your local computer to the gallery, and uses it as the category image.  <br/><br/><span class="btn">Select from Gallery</span> - Prompts you to choose an existing image from the gallery. <br/><br/>![]({% link images/images-ee/icon-pb-camera.png %}){: width="30px"}  - Either drag an image file to the camera tile, or browse to the image and select it from your local file system.|
```
Comment (hickskenh) [inline src/catalog/categories-content-settings.md:53]:
```suggestion
|Category Image|Store View|Specifies an image for the top of the category page. Methods: <br/><br/><span class="btn">Upload</span> - Uploads an image file from your local computer to the gallery and uses it as the category image.  <br/><br/><span class="btn">Select from Gallery</span> - Prompts you to choose an existing image from the gallery. <br/><br/>![]({% link images/images-ee/icon-pb-camera.png %}){: width="30px"}  - Either drag an image file to the camera tile or browse to the image and select it from your local file system.|
```

File: src/catalog/product-content.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Content
+title: Product Content
 ---
 
-The Content section is used to enter and edit the main product description that appears on the product page. The short description can be used in most RSS feeds, and might also appear in catalog listings, depending on theme.
+The _Content_ section is used to enter and edit the main product description that appears on the product page. The short description can be used in most RSS feeds, and might also appear in catalog listings, depending on theme.
```
Comment (hickskenh) [inline src/catalog/product-content.md:5]: if there is a relevant Theme topic, suggest linking "theme" so they can get info on the theme settings.

**Review by hickskenh — APPROVED** [review]
few minor suggestions.  Approved after your review.

File: src/catalog/product-content.md (hunk 2 above)
Comment (leslietilling) [reply src/catalog/product-content.md:5]: Good suggestion. I will looks and see if we have a good one for linking.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**